### PR TITLE
edit cmd/cloud-controller-manager/main.go to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/cloud-controller-manager/main.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/cloud-controller-manager/main.go [hunk 1]
```diff
@@ -75,17 +75,17 @@ func main() {
 
 	cloud, err := cloudprovider.InitCloudProvider(cloudProviderName, c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile)
 	if err != nil {
-		klog.Fatalf("Cloud provider could not be initialized: %v", err)
+		klog.ErrorS("Cloud provider could not be initialized: %v", err)
```
Comment (cheftako) [inline cmd/cloud-controller-manager/main.go:78]: Please take a look at https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/ fo details on using structured logging. Also I believe err should be the first parameter to ErrorS.

File: cmd/cloud-controller-manager/main.go [hunk 2]
```diff
@@ -75,17 +75,17 @@ func main() {
...
 	}
 	if cloud == nil {
-		klog.Fatalf("cloud provider is nil")
+		klog.ErrorS("cloud provider is nil")
```
Comment (cheftako) [inline cmd/cloud-controller-manager/main.go:81]: If your not going to pass Parameters the I wouldn't use a structured parameter call.

File: cmd/cloud-controller-manager/main.go [hunk 3]
```diff
@@ -75,17 +75,17 @@ func main() {
...
 	}
 
 	if !cloud.HasClusterID() {
 		if c.ComponentConfig.KubeCloudShared.AllowUntaggedCloud {
-			klog.Warning("detected a cluster without a ClusterID.  A ClusterID will be required in the future.  Please tag your cluster to avoid any future issues")
+			klog.InfoS("detected a cluster without a ClusterID.  A ClusterID will be required in the future.  Please tag your cluster to avoid any future issues")
 		} else {
-			klog.Fatalf("no ClusterID found.  A ClusterID is required for the cloud provider to function properly.  This check can be bypassed by setting the allow-untagged-cloud option")
+			klog.ErrorS("no ClusterID found.  A ClusterID is required for the cloud provider to function properly.  This check can be bypassed by setting the allow-untagged-cloud option")
```
Comment (yangjunmyfm192085) [inline cmd/cloud-controller-manager/main.go:88]: missing os.Exit(1)

**Pull request closed without merging**